docs/developer/app-store/apps/search.mdx

```diff
@@ -0,0 +1,267 @@
+---
+title: Search
+sidebar_position: 5
+---
+
+import { AppMetadata } from "/components/AppMetadata/AppMetadata.tsx";
+
+<AppMetadata
+  minSaleorVersion="3.10"
+  roadmapUrl="https://github.com/saleor/apps/labels/App%3A%20Search"
+  githubUrl="https://github.com/saleor/apps/tree/main/apps/search"
+/>
+
+## Introduction
+
+The application provides integration with [Algolia](https://www.algolia.com/) search engine. It allows you to index your products and search them using Algolia's API.
+
+## Features
+
+- Synchronize the whole catalog with Algolia
+- Update Algolia index on product changes
+
+## Assumptions & limitations
+
+- Initial synchronization has to be done manually in the App
+- The application creates a separate record for each variant
+- Each of the channels is indexed separately
+- At this moment translations are not supported
+- The exact state of the product quantity is not indexed, only the boolean value (in stock / out of stock)
+- The app will synchronize product variants on the following [events](https://docs.saleor.io/docs/3.x/developer/extending/apps/asynchronous-webhooks):
+  - `PRODUCT_CRETED`
+  - `PRODUCT_UPDATED`
+  - `PRODUCT_DELETED`
+  - `PRODUCT_VARIANT_CREATED`
+  - `PRODUCT_VARIANT_UPDATED`
+  - `PRODUCT_VARIANT_DELETED`
+  - `PRODUCT_VARIANT_OUT_OF_STOCK`
+  - `PRODUCT_VARIANT_BACK_IN_STOCK`
+
+## Glossary
+
+- Algolia record: single entry in the Algolia index. Each Product Variant is a separate record.
+- Index: a collection of Algolia records. We are creating separate indices for each of the channels.
+
+## Permissions required by the app
+
+- `MANAGE_PRODUCTS`
+- `MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES`
```

Comment:
@krzysztofwolski lets add why these permissions are required. It will help to understand scope of the app. 

MANAGE_PRODUCTS is clear to me (but likely not for everyone), but I don't know why `MANAGE_PRODUCT_TYPES_AND_ATTRIBUTES` is required